Upstream-Status: Pending

automake 1.12 has depricated automatic de-ANSI-fication support

this patch avoids these kinds of errors:

| configure.in:2240: error: automatic de-ANSI-fication support has been removed
| Makefile.am:28: error: automatic de-ANSI-fication support has been removed

Signed-Off-By: Nitin A Kamble <nitin.a.kamble@intel.com>
2012/05/02

Index: gmp-5.0.4/configure.in
===================================================================
--- gmp-5.0.4.orig/configure.in
+++ gmp-5.0.4/configure.in
@@ -59,7 +59,7 @@ dnl
 dnl  Note that there's a copy of these options in the top-level Makefile.am,
 dnl  so update there too if changing anything.
 dnl
-AM_INIT_AUTOMAKE([1.8 gnu no-dependencies $(top_builddir)/ansi2knr])
+AM_INIT_AUTOMAKE([1.8 gnu no-dependencies])
 AM_CONFIG_HEADER(config.h:config.in)
 AM_MAINTAINER_MODE
 
@@ -2236,9 +2236,6 @@ fi
 echo "      MPN_PATH=\"$path\""
 
 
-# Automake ansi2knr support.
-AM_C_PROTOTYPES
-
 CL_AS_NOEXECSTACK
 
 GMP_PROG_AR
Index: gmp-5.0.4/Makefile.am
===================================================================
--- gmp-5.0.4.orig/Makefile.am
+++ gmp-5.0.4/Makefile.am
@@ -25,7 +25,7 @@
 # Makefiles in subdirectories, but here we must omit it so automake gives
 # the actual ansi2knr build rule, not "cd $(top_builddir) && make ansi2knr".
 #
-AUTOMAKE_OPTIONS = 1.8 gnu no-dependencies ansi2knr
+AUTOMAKE_OPTIONS = 1.8 gnu no-dependencies
 
 
 # Libtool -version-info for libgmp.la and libmp.la.  See "Versioning" in the
